stories/examples/Ratings.js

```diff
@@ -0,0 +1,164 @@
+    rating={2.35}
+    total= {'20 Ratings'}
+    backgroundColor={'royalblue'}
+    theme={RatingBarTheme}
+    viewBox= "0 0 51 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
+  />
+)
+
+export const FullRatings = (
+  <RatingBar
+    ratingScale={5}
+    rating={5}
+    total= {'15'}
+    backgroundColor={'red'}
+    theme={RatingBarTheme}
+    viewBox="106 -228 512 512"
+    stroke="black"
+    strokeWidth="20"
+    itemHeight="300px"
+    itemWidth="300px"
+    pathD= {pathD.SETTINGS}
+  />
+)
+
+export const FourRatings = (
+  <RatingBar
+    ratingScale={5}
+    rating={4}
+    total= {'200 Rated'}
+    backgroundColor={'yellow'}
+    theme={RatingBarTheme}
+    viewBox="106 -228 512 512"
+    stroke="black"
+    strokeWidth="25"
+    itemHeight="200px"
+    itemWidth="200px"
+    pathD= {pathD.SEARCH}
+  />
+)
+
+export const ThreeRatings = (
+  <RatingBar
+    ratingScale={5}
+    rating={3}
+    total= {'27 Awesome'}
+    backgroundColor={'green'}
+    theme={RatingBarTheme}
+    viewBox="0 0 42 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= {pathD.AC}
+  />
+)
+
+export const TwoRatings = (
+  <RatingBar
+    ratingScale={5}
+    rating={2}
+    total= {'403'}
+    backgroundColor={'brown'}
+    theme={RatingBarTheme}
+    viewBox= "0 0 51 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
+  />
+)
+
+export const OneRatings = (
+  <RatingBar
+    ratingScale={5}
+    rating={1}
+    total= {'3333'}
+    backgroundColor={'pink'}
+    theme={RatingBarTheme}
+    viewBox= "0 0 51 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
+  />
+)
+
+export const ManyRatings = (
+  <RatingBar
+    ratingScale={63}
+    rating={34}
+    total= {'150'}
+    backgroundColor={'gold'}
+    theme={RatingBarTheme}
+    viewBox= "0 0 51 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
+  />
+)
+
+export const Partial1 = (
+  <RatingBar
+    ratingScale={5}
+    rating={3.5}
+    total= {'150'}
+    backgroundColor={'royalblue'}
+    theme={RatingBarTheme}
+    viewBox= "0 0 51 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
+  />
+)
+
+export const Partial2 = (
+  <RatingBar
+    ratingScale={5}
+    rating={4.4}
+    total= {'150'}
+    backgroundColor={'royalblue'}
+    theme={RatingBarTheme}
+    viewBox= "0 0 51 48"
+    stroke="black"
+    strokeWidth="2"
+    itemHeight="25px"
+    itemWidth="25px"
+    pathD= "m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
```

Comment:
Well, to a certain extent yes.  That means 1) you'd always have to have the path data and 2) hope that the svg isn't complex to a point that it needs more than a path.